pkgs/development/python-modules/wled/default.nix

```
@@ -36,10 +42,17 @@ buildPythonPackage rec {
     pytestCheckHook
   ];
 
+  postPatch = ''
+    # Upstream doesn't set a version for the pyproject.toml
+    substituteInPlace pyproject.toml \
+      --replace "0.0.0" "${version}" \
+      --replace "--cov" ""
+  '';
+
   pythonImportsCheck = [ "wled" ];
 
   meta = with lib; {
-    description = "Asynchronous Python client for WLED";
+    description = "Python client for WLED";
```

https://github.com/frenck/python-wled/blob/v0.5.0/pyproject.toml#L4

---

```suggestion
    description = "Asynchronous Python client for WLED";
```

@fabaff I think we want to keep the original description here.